warehouse/ingest-scripts/src/main/resources/bin/ingest/generate-accumulo-config-cache.sh

```diff
@@ -12,12 +12,11 @@ cd $THIS_DIR
 
 export ACCUMULO_OTHER_OPTS="${ACCUMULO_OTHER_OPTS} $HADOOP_INGEST_OPTS"
 
-ADDJARS="$( cd ../../lib && pwd )/*"
-
-CLASSPATH=$ADDJARS $WAREHOUSE_ACCUMULO_HOME/bin/accumulo \
+export CLASSPATH
+$WAREHOUSE_ACCUMULO_HOME/bin/accumulo \
   jar ${DATAWAVE_INGEST_CORE_JAR} datawave.ingest.config.TableConfigCacheGenerator \
-  -u $USERNAME \
-  -p $PASSWORD \
-  -i $WAREHOUSE_INSTANCE_NAME \
-  -zk $WAREHOUSE_ZOOKEEPERS \
+  -user $USERNAME \
+  -pass $PASSWORD \
+  -instance $WAREHOUSE_INSTANCE_NAME \
+  -zookeepers $WAREHOUSE_ZOOKEEPERS \
```

Comment:
These changes are because TableConfigCacheGenerator uses [OptionsParser](https://github.com/NationalSecurityAgency/datawave/blob/master/warehouse/ingest-core/src/main/java/datawave/ingest/OptionsParser.java) and it needs different arguments. I didn't track down when that changed and it broke the args.